src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -1248,9 +1248,40 @@ public QueryOperationResponse<T> LoadProperty<T>(object entity, string propertyN
         public Uri GetReadStreamUri(object entity)
         {
             Util.CheckArgumentNull(entity, "entity");
-            EntityDescriptor box = this.entityTracker.GetEntityDescriptor(entity);
+            EntityDescriptor box = GetEntityDescriptorForStreamEntity(entity);

```

Comment:
I believe the style is to append ```this.``` and looking at the build errors, stylecop would have caught this.